plugin/plugin.go

```diff
@@ -171,8 +243,10 @@ func Exec(ctx context.Context, args Args) (err error) {
 	}
 	if args.JsonSecret != "" {
 		jsonData := JsonOutput{
-			Token: tokenData,
-			Jwt:   jwtSigned,
+			Token:           tokenData,
+			Jwt:             jwtSigned,
+			RepositoryCount: len(tokenData.Repositories),
+			Permissions:     tokenData.Permissions,
```

Comment:
```suggestion
			Token:           tokenData,
			Jwt:             jwtSigned,
```
this information is already included in `token` and thus not needed